hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/record/RecordMigrationInfo.java

```diff
@@ -27,6 +27,14 @@
     private Data key;
```

Comment:
I don't know how often the RecordMigrationInfo is send; but it is often I would consider implements IdentifiedDataSerializable.